### PR TITLE
List separators

### PR DESCRIPTION
**Pull request by Kiarokh**
fix https://github.com/Lundalogik/lime-elements/issues/2512

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/list/list-item.types.ts [hunk 1]
```diff
@@ -49,6 +49,8 @@ export interface ListItem<T = any> {
 
 export interface ListSeparator {
     separator: true;
+
+    separatorTitle?: string;
```
Comment (jgroth) [inline src/components/list/list-item.types.ts:53]: It might make sense to just name this `text` as for regular `ListItem`s

Comment (jgroth) [reply src/components/list/list-item.types.ts:53]: It should at least not have `separator` in the name, we already know that it's a separator so we do not need to state that again :slightly_smiling_face:

Comment (Kiarokh) [reply src/components/list/list-item.types.ts]: ⚡

File: src/components/list/examples/list-separator.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { ListItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
+/**
+ * List with separators
+ *
+ * Separators are a simple yet powerful design elements that can be
```
Comment (john-traas) [inline src/components/list/examples/list-separator.tsx:7]:
```suggestion
 * Separators are simple yet powerful design elements that can be
```

**Pull request merged**